content/300-guides/050-database/880-supabase.mdx

```diff
@@ -28,8 +28,46 @@ Many aspects of using Prisma with Supabase are just like using Prisma with any o
 - use [`db push`](/concepts/components/prisma-migrate/db-push) to push changes in your schema to Supabase
 - use [Prisma Client](/concepts/components/prisma-client) in your application to talk to the database server at Supabase
 
+## Specific considerations
+
+If you'd like to use the [connection pooling feature](https://supabase.com/docs/guides/integrations/prisma#connection-pooling-with-supabase) available with Supabase, you will
+need to add `pgbouncer=true` to the end of your `DATABASE_URL` string.
+
+```env file=.env
+# Connect to Supabase with PgBouncer.
+DATABASE_URL="postgres://postgres:__PASSWORD__@db.__YOUR_SUPABASE_PROJECT__.supabase.co:6543/postgres?pgbouncer=true"
+```
+
+If you would like to use the Prisma CLI in order to perform other actions on your database (e.g. migrations) you will need to add a `DIRECT_URL` string so that the CLI can bypass PgBouncer:
```

Comment:
Sorry to be nitpicky here, but `DATABASE_URL` and `DIRECT_URL` are just random env var strings, they could also be `FOO` and `BAR`. The relevant bits are `url` and `directUrl` in the `datasource` block.

You could write:
```
... to the end of the `DATABASE_URL` environment variable used in the `datasource.url` property:
```
and equivalent for the direct URL.

Reply:
I appreciate the feedback! I'll clarify my language here.